src/components/PupilViews/PupilReview/PupilReview.view.tsx

```diff
@@ -51,6 +68,10 @@ export const PupilViewsReview = (props: PupilViewsReviewProps) => {
     sectionResults,
   );
 
+  const pupilClient = useOakPupil();
+  const { logAttempt } = pupilClient;
+  const isShowShareButton = useFeatureFlagEnabled("share-results-button");
```

Comment:
This will be for share and printable results so I'd rename to a plural isShowShareButtons